reference/docs-conceptual/whats-new/What-s-New-in-PowerShell-Core-61.md

```diff
@@ -37,6 +37,12 @@ so that any modules or scripts that use these APIs can rely on them being availa
 
 The Windows Compatibility Pack enables PowerShell Core to use **more than 1900 cmdlets that ship with Windows 10 October 2018 Update and Windows Server 2019**.
 
+## Support for Application Whitelisting
+
+PowerShell Core 6.1 has parity with Windows PowerShell 5.1 supporting [AppLocker](https://docs.microsoft.com/en-us/windows/security/threat-protection/windows-defender-application-control/applocker/applocker-overview)
+and [Device Guard](https://docs.microsoft.com/en-us/windows/security/threat-protection/device-guard/introduction-to-device-guard-virtualization-based-security-and-windows-defender-application-control) application whitelisting.
```

Comment:
"DeviceGuard has been re-branded to "Windows Defender Application Control".  This is annoying but I think we should mention both titles.

Reply:
Looks like this was merged.  Please submit new PR, your feedback makes sense.